modules/exploits/multi/http/mantisbt_manage_proj_page_rce.rb

```diff
@@ -0,0 +1,123 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Mantis v1.1.3 Remote Code Execution',
```

Comment:
The module name typically does not contain the software version, instead favoring describing the vulnerably functionality, such as page name or parameter name.

Also, `remote` is implied, by virtue of living in `modules/exploits/multi/http`.

Consider something like this:

`Mantis manage_proj_page PHP Code Execution`